manage.py

```diff
@@ -12,4 +12,8 @@
             "available on your PYTHONPATH environment variable? Did you "
             "forget to activate a virtual environment?"
         ) from exc
+    if 'runserver' in sys.argv:
```

Comment:
@KicaRonaldOkello , this should be removed. It has a security risk

Reply:
will remove it

Reply:
will remove it